modules/virt-configuring-cluster-dpdk.adoc

```diff
@@ -87,7 +87,7 @@ The compute nodes automatically restart after you apply the `MachineConfigPool`
 $ oc get performanceprofiles.performance.openshift.io profile-1 -o=jsonpath='{.status.runtimeClass}{"\n"}'
 ----
 
-. Set the previously obtained `RuntimeClass` name as the default container runtime class for the `virt-launcher` pods by editing the `HyperConverged` custom resource (CR):
+. Set the previously obtained `RuntimeClass` name as the default container runtime class for the `virt-launcher` pods by editing the `HyperConverged` CR:
```

Comment:
Because this is the first instance of "CR", I would suggest keeping it as it was previously, spelling out custom resource (CR)